README.md

```diff
@@ -25,12 +25,22 @@ yarn add @fullstory/server-api-client
 
 #### Initializing the Client
 
-Use the `init` function to initialize the FullStory client with [your API key](https://developer.fullstory.com/server/v2/authentication/).
+Use the `init` function to initialize the FullStory client with [your API key](https://developer.fullstory.com/server/v2/authentication/). The `FullStoryOptions` used in `init` function will be used for all server API requests, unless overridden by [per request options](#request-options).
```

Comment:
Go ahead and remove the statement about per-request overrides given our comment thread elsewhere in this PR.

```suggestion
Use the `init` function to initialize the FullStory client with [your API key](https://developer.fullstory.com/server/v2/authentication/). The `FullStoryOptions` used in `init` function will be used for all server API requests.
```